Signal/src/call/CallService.swift

```diff
@@ -260,22 +252,24 @@ protocol CallServiceObserver: class {
 
             self.peerConnectionClient = peerConnectionClient
 
-            return self.peerConnectionClient!.createOffer()
-        }.then(on: CallService.signalingQueue) { (sessionDescription: HardenedRTCSessionDescription) -> Promise<Void> in
```

Comment:
A lot of whitespace was fixed. If you want, I'll go back and revert this. I know you're not a fan.